helm/ingress-controller/templates/controller-deployment.yaml

```diff
@@ -45,12 +45,19 @@ spec:
         {{- if .Values.region }}
         - --region={{ .Values.region}}
         {{- end }}
+        {{- if .Values.serverAddr }}
+        - --server-addr={{ .Values.serverAddr}}
+        {{- end }}
         - --health-probe-bind-address=:8081
         - --metrics-bind-address=:8080
         - --election-id={{ include "ngrok-ingress-controller.fullname" . }}-leader
         securityContext:
           allowPrivilegeEscalation: false
         env:
+        {{- range $key, $value := .Values.extraEnv }}
```

Comment:
nit: Can we move this to the end of the `env` section since its a last value wins?

Reply:
we can, i think i did this because it made the test case simpler :) i'll move it to the bottom and see if i can find a way to test it without hardcoding the number of other envs